app/views/user_mailer/_requested.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Your <%= link_to 'reservation request', reservation_url(@reservation, only_path: false) %> has been approved!<p>
+
+<p>You can pick up your <%= @reservation.equipment_model.name %> at <%= @reservation.start_date.to_date.to_s(:long) %>. Remember to bring it back on <%= @reservation.due_date.to_date.to_s(:long) %>!
```

Comment:
you can take out all the `to_date`s, since the `start_date` and `due_date`s are both `Date`s.